.github/workflows/push-production.yaml

```diff
@@ -182,6 +182,8 @@ jobs:
       - uses: chainguard-dev/actions/inky-build-pkg@main
         with:
           package-name: patch
+          empty-workspace: false
```

Comment:
This isn't needed anymore, the action now just creates an empty directory if one didn't already exist.